packages/ra-ui-materialui/src/field/ReferenceArrayField.tsx

```diff
@@ -16,6 +15,7 @@ import {
 import { fieldPropTypes, PublicFieldProps, InjectedFieldProps } from './types';
 import { ClassesOverride } from '../types';
 import sanitizeFieldRestProps from './sanitizeFieldRestProps';
+import { LinearProgress } from '../layout';
```

Comment:
useless change

Reply:
Nope, we are now using our own LinearProgress instead of MUI